### DISCUSSION TITLE
blog/blogdown-netlify-new-post-workflow/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#97] [Comments]
# blog/blogdown-netlify-new-post-workflow/

A workflow for writing new posts built around GitHub pull requests and Netlify deploy previews.

https://www.garrickadenbuie.com/blog/blogdown-netlify-new-post-workflow/#publish

**Comment by andrewpbray** [comment]
Very handy post - thanks!

I found my way here while trying to use the `publishdate` field to act as a scheduler. Unfortunately, as you note, this tool chain is only capable of being a "scheduler". 

My goal to pre-populate a course repo with assignments, lecture materials, etc, (with `publishdate`s) and have them unspool to the website over the course of the semester according to the course calendar. The missing link is triggering netlify to re-render after the publish date. Do you have any ideas for using GHA to schedule the re-render?

**Reply by andrewpbray** [reply]
I may have found a straight forward solution here:

[https://oliverjam.es/blog/schedule-netlify-github-actions/](https://oliverjam.es/blog/schedule-netlify-github-actions/)

The idea is to set up a cron job through github actions that triggers a build hook at netlify. Seems to work for me.

**Reply by gadenbuie** [reply]
Thanks @andrewpbray! That link is excellent and is the perfect way to automate the site rebuilding. (I kind of wonder why Netlify doesn't offer a similar option?)
